flashcards/backend/docs/install-poetry.rst

```diff
@@ -0,0 +1,7 @@
+install\-poetry module
```

Comment:
We are not documenting thiedeman party scripts. It is not out job😉

Reply:
Done.